changelogs/fragments/7874-incus_connection_treats_inventory_hostname_as_literal_in_remotes.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "incus connection plugin - treats ``inventory_hostname`` as a variable instead of a literal in remote connections.  (https://github.com/ansible-collections/community.general/issues/7874)."
```

Comment:
```suggestion
  - "incus connection plugin - treats ``inventory_hostname`` as a variable instead of a literal in remote connections (https://github.com/ansible-collections/community.general/issues/7874)."
```